sindresorhus/refined-github
# Improve feature application on AJAXed widgets

# Tests

1. Visit an issue or PR (like this one)
2. Add a reaction

Or:

1. Visit an issue or PR (like this one)
2. Edit a comment

You should still see the features being applied correctly:

- `sticky-discussion-sidebar`
- `clean-sidebar`
- `reaction-avatars`
- `edit-comments-faster`’s icon (fixes https://github.com/sindresorhus/refined-github/issues/2572)
- `mark-unread`’s button

Follows https://github.com/sindresorhus/refined-github/pull/2533 https://github.com/sindresorhus/refined-github/pull/2344
